src/Nethermind/Nethermind.Cli/NodeManager.cs

```diff
@@ -71,7 +71,8 @@ public async Task<JsValue> PostJint(string method, params object[] parameters)
                 stopwatch.Start();
                 object result = await _currentClient.Post<object>(method, parameters);
                 stopwatch.Stop();
-                Console.WriteLine($"Request complete in {stopwatch.ElapsedMilliseconds}ms");
+                decimal totalMicroseconds = stopwatch.ElapsedTicks * (1_000_000m / Stopwatch.Frequency);
+                Console.WriteLine($"Request complete in {totalMicroseconds}μs");
```

Comment:
Is it better than built in stuff in stopwatch? Maybe lets have it as a extension method?